presto-product-tests-utils/pom.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>presto-root</artifactId>
+        <groupId>com.facebook.presto</groupId>
+        <version>0.116-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>presto-product-tests-utils</artifactId>
+    <name>presto-product-tests-utils</name>
+
+    <properties>
+        <air.main.basedir>${project.parent.basedir}</air.main.basedir>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.facebook.presto</groupId>
+            <artifactId>presto-main</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.slf4j</groupId>
+                    <artifactId>slf4j-jdk14</artifactId>
```

Comment:
In this patch we used logback (through logback-classic.jar) as slf4j backend. I tried to replace it with java.util.logging backend but there is a problem with MDC. We store test name in MDC map and then use it in logging pattern. We could log thread id and and then do the matching by it, but it is less user friendly. So I would opt for keeping logback as logging backend here. This should not be a problem as this is not part of the product.

Api is the other thing. If you prefer I can change logging statements throughout presto-product-tests module from slf4j to java.util.logging.

Based on your diagram it looks like presto is using logback as logging backend. But it seems to me that it is not the case. Am I missing something?